rythmbadge/request_handler_lib.c

```diff
@@ -351,7 +351,7 @@ static void process_receive_notification(void * p_event_data, uint16_t event_siz
 	request_event.request_timepoint_ticks = timepoint_ticks;
 
 	
-	NRF_LOG_INFO("REQUEST_HANDLER: Which request type: %u, Ticks: %u\n", request_event.request.which_type, request_event.request_timepoint_ticks);
```

Comment:
Undo this change? Isn't this an useful log message to have? Same for the other two below